src/web/src/utils.js

```diff
@@ -210,6 +210,22 @@ export const addDays = (date, days) => {
   return result;
 };
 
+/**
+ * Export Course Data Given CRN Value.
+ */
+export const conflictCourse = (allCourseData, inputCRNValue) => {
+  return allCourseData.find((currentCourse) => {
+    //Search For Desired CRN In Current Course.
+    let isFound = currentCourse.sections.some((currentSection) => {
+      return currentSection.crn === inputCRNValue;
+    });
+    //Found Course w/ Appropriate CRN.
+    if (isFound) {
```

Comment:
js tip: you could just return `isFound` instead of checking it and returning `currentCourse`, so something like this `allCourseData.find((currentCourse) => currentCourse.sections.some((currentSection) => currentSection.crn === inputCRNValue))`

Reply:
Hello Josh. Thank you for this suggestion. However, I just tried this out and unfortunately it did not work. The reason I actually implemented it this way is due to the fact that .some() returns True/False and we are interested in the corresponding currentCourse that made it return True for .some().